tools/docker.sumologic-powershell-sdk/readme.md

```diff
@@ -0,0 +1,48 @@
+# docker container to run sumologic powershell sdk on .net core /linux
```

Comment:
> Docker container to run Sumo Logic PowerShell SDK with .NET Core on Linux

Not a big problem but in document (here and later paragraph) it's better mention the brands/productions as it's well-known form
as
`PowerShell`, `.NET Core`, `Linux`, `Docker`, `Sumo Logic`, etc.